python/grass/pygrass/raster/__init__.py

```diff
@@ -29,6 +23,17 @@
 from grass.pygrass.raster.segment import Segment
 from grass.pygrass.raster.rowio import RowIO
 
+
+def import_grass_libs():
+    import grass.lib.gis as libgis
+    libgis.G_gisinit("")
+    import grass.lib.raster as libraster
+    import grass.lib.rowio as librowio
+    return libgis, libraster, librowio
+
+
+libgis, libraster, librowio = import_grass_libs()
```

Comment:
Using that pattern might complicate static analysis, as the imports aren't directly visible (it is a kind of lazy loading, but the call to that function is made during a parsing of the file for an import for example).

Does the lib raster and lib rowio import successfully and work correctly if `libgis.G_gisinit("")` isn't called strictly before, but underneath?

This pattern seems to be on multiple files.